simplified-links-json/src/main/java/org.nypl.simplified.links.json/LinkSerialization.kt

```diff
@@ -17,7 +17,7 @@ object LinkSerialization {
    */
 
   fun serializeLink(link: Link): ObjectNode {
-    val node = this.mapper.createObjectNode()
+    val node = mapper.createObjectNode()
```

Comment:
Why'd you remove `this` here? Not complaining just curious :)

Reply:
Oof. Not intentional. Suspect that may've been an IDE issue.